docs/manual/creating-a-checker.tex

```diff
@@ -1926,7 +1927,7 @@
 java -Xbootclasspath/p:$CHECKERFRAMEWORK/checker/dist/javac.jar \
   -cp $CHECKERFRAMEWORK/checker/dist/checker.jar \
   org.checkerframework.dataflow.cfg.CFGVisualizeLauncher \
-  MyClass.java output/ -class MyClass -method test -pdf
+  MyClass.java -output output/ -class MyClass -method test -pdf
```

Comment:
To simplify the command, how about leaving off the `-output` command-line argument?

Reply:
Thanks for the comments!
If we remove `--output`, then the `outputdir` becomes a mandatory argument (`String outputdir = args[1];`). Although users want to generate a string graph, they still need to provide `outputdir` in the command. E.g., `MyClass.java output/ -class MyClass -method test -string`.

Reply:
Sorry that my comment was ambiguous.  I think that the tool should support the command-line argument.  I was just suggesting that this example invocation could leave it off, using the default, to make the command line shorter and to reduce the difference between the `--string` version below.

Reply:
Oh! I get it. Thanks a lot! I will do it.